test/e2e/specs/site-editor/new-templates-list.spec.js

```diff
@@ -50,7 +50,7 @@ test.describe( 'Templates', () => {
 		} );
 		await admin.visitSiteEditor( { path: '/wp_template/all' } );
 		// Global search.
-		await page.getByRole( 'searchbox', { name: 'Filter list' } ).click();
+		await page.getByRole( 'searchbox', { name: 'Search' } ).click();
 		await page.keyboard.type( 'tag' );
```

Comment:
It's probably better to use [`fill()`](https://playwright.dev/docs/api/class-locator#locator-fill) for similar cases unless we specifically test typing in the input. It will handle input focus for us.

```js
await page.getByRole( 'searchbox', { name: 'Search' } ).fill( 'tag' );
```

Reply:
Thank you for the advice. Updated 👍